src/app/routes.tsx

```diff
@@ -59,6 +61,12 @@ const routes: AppRouteConfig[] = [
     label: 'Azure',
     path: '/browser/Azure',
   },
+  {
+    component: <ImageDetails title='Cloud Image Directory | Image Details'/>,
+    exact: true,
+    label: 'Search',
+    path: '/image/:provider/:region/:imageName',
```

Comment:
Does it make sense to change this to browser?
```suggestion
    path: '/browser/:provider/:region/:imageName',
```